### ISSUE TITLE
accessibilitySupport should be array of Concept only

### ISSUE THREAD

**Issue by nickevansuk** [openactive/modelling-opportunity-data#195]
In line with our restriction of `activity`, `accessibilitySupport` should be an array of `Concept` and not have the option to be an array of `string`.

**Comment by ldodds** [comment]
Originally we had all properties like this using Concepts only, but based on feedback we moved it back a bit.

As changing would potential break feeds that now conform to 2.0, a measured approach would be to:

* update the data validator to recommend use of Concepts and not strings and indicate that future release will change this
* update the spec to say that this may change in future and more strongly recommend Concept over strings
* provide a link to the preferred scheme for people to use for these terms
